src/appkit.cs

```diff
@@ -9002,6 +9023,11 @@ partial interface NSOutlineViewDelegate {
 
 		[Export ("outlineView:didRemoveRowView:forRow:")]
 		void DidRemoveRowView (NSOutlineView outlineView, NSTableRowView rowView, nint row);
+
+		[Mac (10,16)]
+		[Export ("outlineView:tintConfigurationForItem:")]
+		[return: NullAllowed]
+		NSTintConfiguration OutlineView (NSOutlineView outlineView, NSObject item);
```

Comment:
`OutlineView` is the name of the first parameter ;) 

```suggestion
		NSTintConfiguration GetTintConfiguration (NSOutlineView outlineView, NSObject item);
```